src/Extensions/StringExtension.ts

```diff
@@ -0,0 +1,22 @@
+declare global {
+    interface String {
+        toBase64(): string;
+        fromBase64(): string;
+    }    
+}
+
+/**
+ * AS STRING
+ * String prototype has this of type `String`, the capitalization causes issues so I have to cast as `string`
+ */
+String.prototype.toBase64 = function (): string {
+    // see AS STRING comment
+    return btoa(this as string);
```

Comment:
Do you want to add the additional uriencode call to ensure that the `=` don't cause problems?

Reply:
I was going to, however URLSearchParams from useQuery that we added and window.open handle that for us